lib/babe/parachain_inherents.go

```diff
@@ -0,0 +1,345 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package babe
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+// signature could be one of Ed25519 signature, Sr25519 signature or ECDSA/SECP256k1 signature.
+type signature [64]byte
```

Comment:
nit do we really need a `signature` type? Can't we just use `[64]byte` directly? I.e. `type collatorSignature [64]byte`, `type validatorSignature [64]byte`, `Signature [64]byte`
Asking since it conflicts with signature import, and `signature` is rather generic and would be nice to use as a variable name here and there.